docs/v0.6.0/0.6.md

```diff
@@ -0,0 +1,21 @@
+# Documentation
+
+## Table of Contents
+
+- [Install Rio](./install.md)
+- [Rio Concepts](./concept.md)
+- [Quick Start](./quick-stack.md)
+- [Running workloads](./workloads.md)
+- [Configuring Router](./router.md)
+- [Configuring PublidDomain And ExternalService](./publicdomain-external-services.md)
+- [Using Riofile](./riofile.md)
+- [Monitoring Workloads](./monitoring.md)
+- [Using AutoScaling](./autoscaling.md)
+- [Configuring Rio with RBAC](./rbac.md)
+- [Continuous Delivery with Rio](./continous-deployment.md)
+- [Using Rio for developing your application](./developing-application-using-rio.md)
+- [Using Dashboard](./dashboard.md)
+- [CLI Reference](./cli-reference.md)
+- [API Reference](./api-docs.md)
+- [Troubleshooting](./faq.md)
+
```

Comment:
Think dax wanted quickstart here, we can do later

Reply:
yeah, we can change the structure later